assets/scss/main.scss

```diff
@@ -5,10 +5,17 @@
 @include carbon.grid-css-grid;
 @include carbon.grid-flex-grid;
 
+@font-face {
```

Comment:
I'm wondering if we need to include the `.ttf` file in our code base, or if we can use a similar approach to what they do in saiba.

![Screenshot 2023-04-10 at 1 17 04 PM](https://user-images.githubusercontent.com/6276074/230990157-0b72a9ea-3d0b-43b8-bb42-276f43ac1638.png)

Reply:
Sure, we can do that. Changed in https://github.com/Qiskit/qiskit.org/pull/3108/commits/d5c8c4739ccfd44328918d8cfc267c0461e54466.